api/package.json

```diff
@@ -15,6 +15,11 @@
     "seed": "knex seed:run --specific=main.js",
     "seed-dev": "NODE_ENV=development DEV_DB_HOST=localhost npm run seed",
     "seed-users": "knex seed:run --specific=base-users.js",
+    "make-mongoose-migrate": "migrate --config mongo-migrate.js create",
```

Comment:
I'm seeing this error when I try to do either this or the `-dev` command
`make-mongoose-migrate`